Issue and PR templates (#2842)
firedrakeproject/firedrake: .github/ISSUE_TEMPLATE/documentation.md

---
name: "\U0001F4DC Documentation"

> description: Suggest an idea for this project

Reviewer: I don't think that this is clear.

Reviewer 2: `Suggest your documentation concern or idea`

> description: Suggest an idea for this project

Reviewer 2: ```suggestion
description: Highlight a documentation issue
```

title: 'DOCS: '
labels: [documentation]
assignees: ''

---

**Describe the issue**
A clear and concise description of what documentation is missing or incorrect.

**Where to find**

> Where the error is seen or where documentation shoud be:

Author: ```suggestion
Where to find the error or where documentation should be:
```

(website/github/docstrings/etc.)

> - Make a list

Reviewer: "Location" instead of "Where to find"? Also typo "should". Just generally a bit unclear.
